kafka/build_darwin.go

```diff
@@ -6,8 +6,8 @@
 package kafka
 
 // #cgo CFLAGS: -I${SRCDIR}
-// #cgo LDFLAGS: ${SRCDIR}/librdkafka/librdkafka_darwin.a  -lm -lsasl2 -lz -ldl -lpthread
+// #cgo LDFLAGS: ${SRCDIR}/librdkafka/librdkafka_darwin.a  -lm -lsasl2 -ldl -lpthread
```

Comment:
i think this is statically linked now in the bundle, right?

Reply:
Huhm, no, this is due to changes with the .pc file.
Good catch!